docs/guides/integration-prebid.md

```diff
@@ -0,0 +1,187 @@
+
+To configure the module to use this mode, you must do **either** of the following:
+
+- Set a cookie named `__uid2_advertising_token` to the advertising token (see [Server-Only Cookie Example](#server-only-cookie-example)).
+
+- Set `value` to an ID block containing the advertising token (see [Server-Only Value Example](#server-only-value-example)).
+
+### Server-Only Cookie Example
+
+This example sets a cookie named `__uid2_advertising_token` to the advertising token.
+
+#### Cookie
+
+The following example uses a fictitious value for the advertising token, to show what the cookie might look like.
+
+```
+__uid2_advertising_token=eb33b0cb-8d35-4722-b9c0-1a31d4064888
+```
+
+#### Configuration
+
+```javascript
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2'
+    }]
+  }
+});
+```
+
+### Server-Only Value Example
+
+This example sets `value` to an ID block containing the advertising token.
+
+Configuration:
+
+```javascript
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2'
+      value: {
+        'uid2': {
+          'id': '...advertising token...'
+        }
+      }
+    }]
+  }
+});
+```
+
+## Storage of Internal Values
+
+The UID2 Prebid module stores some internal values. By default, all values are stored in HTML5 local storage. If needed, you can switch to cookie storage by setting `params.storage` to `cookie`. The cookie size can be significant, so we don't recommend this solution, but it is a possible solution if local storage is not an option.
+
+## Sample Token
+
+The following sample is fictitious, but shows what the token response object looks like:
+
+```javascript
+{
+  "advertising_token": "...",
+  "refresh_token": "...",
+  "identity_expires": 1633643601000,
+  "refresh_from": 1633643001000,
+  "refresh_expires": 1636322000000,
+  "refresh_response_key": "wR5t6HKMfJ2r4J7fEGX9Gw=="
+}
+```
+
+## Prebid Implementation Notes and Tips
+
+In planning your Prebid implementation, consider the following:
+
+- If you're trying to limit the size of cookies, provide the token in configuration and use the default option of local storage.
+
+- If you provide an expired identity, and the module has a valid identity which was refreshed from the identity you provide, the module uses the refreshed identity. The module stores the original token that it used for refreshing the token, and uses the refreshed tokens as long as the original token matches the token that you provided.
+
+- If you provide a new token that does not match the original token used to generate any refreshed tokens, all stored tokens are discarded and the new token used instead (refreshed if necessary).
+
+- During integration testing, you can set `params.uid2ApiBase` to `"https://operator-integ.uidapi.com"`. Be aware that you must use the same environment (production or integration) that you use for generating tokens.
```

Comment:
Is there a specific reason why you didn't copy 
__Parameter Descriptions for the usersync Configuration_ Section_ Section from the [readme]( https://github.com/prebid/Prebid.js/blob/master/modules/uid2IdSystem.md) to here given you have copied all other sections?

Reply:
@sunnywu  Really not sure. But, I've added it.

Reply:
Not at all sure but I've added it now.